docs/source/posts/2022/2022-06-29-week-4-praneeth.rst

```diff
@@ -0,0 +1,53 @@
+==================================
+Week 4 - Fixing the Clamping Issue
+==================================
+
+.. post:: June 29 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
```

Comment:
Maybe you can rewrite the last line for more clarity. You can write it like this `to clamp all the shapes successfully.`